### ISSUE TITLE
Contrast Issue of placeholder in searchbar

### ISSUE THREAD

**Issue by ayanchoudhary** [cc-archive/vocabulary-legacy#473]
## Problem

![84492414-c744da80-acc3-11ea-88de-40c00725f888](https://user-images.githubusercontent.com/45792768/84812468-15dbd700-b02c-11ea-9d48-a1d3d0dcd3db.png)

## Description

Currently, this span has a contrast ratio of **3.45** whereas the desired minimum contrast ratio should be **4.5** or **AA/AAA large**

## Implementation

- [ ] I would be interested in implementing this feature.

**Comment by ayanchoudhary** [comment]
@panchovm @dhruvkb

**Comment by zackkrida** [comment]
Closing this issue, as it's a bit of a catch-22. If we _increase_ contrast of the placeholder, it will be mistaken by users for a _value_ instead of a placeholder, which is itself an accessibility concern. Our `aria` labels should be sufficient for screen-reader users.

I spot checked a couple of popular websites, and think this will be okay. GitHub's inputs, for example, use quite a light label too:

<img width="292" alt="Screen Shot 2020-09-23 at 6 49 30 AM" src="https://user-images.githubusercontent.com/6351754/94043789-0fff3b80-fd69-11ea-89db-ef273b5e3c2c.png">
